### PR TITLE
issue  #15 fix

### PR DESCRIPTION
**Pull request by Nic30**

### CODE REVIEW

File: runtime/Cpp/runtime/src/atn/LexerMoreAction.cpp [hunk 1]
```diff
@@ -38,7 +38,11 @@ using namespace antlr4;
 using namespace antlr4::atn;
 using namespace antlr4::misc;
 
-const Ref<LexerMoreAction> LexerMoreAction::INSTANCE { new LexerMoreAction() };
+const Ref<LexerMoreAction> LexerMoreAction::getInstance() {
+	static Ref<LexerMoreAction> instance = std::shared_ptr<LexerMoreAction>(
```
Comment (nuuman) [inline runtime/Cpp/runtime/src/atn/LexerMoreAction.cpp]: Shouldn't this be `static Ref<LexerMoreAction> instance = Ref<LexerMoreAction>` for consistency? 

Same goes for numerous places.

Comment (Nic30) [inline runtime/Cpp/runtime/src/atn/LexerMoreAction.cpp]: I would like to use const Ref<T> T::INSTANCE = std::make_shared<T>();
as well but () constructors are private.

**Comment by mike-lischke** [comment]
The patch is just fine as is. I just had to fix formatting a bit and simplified init of the static vars. Thanks.

**Comment by mike-lischke** [comment]
@Nic30, in order to accept your patch I have to ask you to sign the contributors list (add yourself to contributors.txt) and send this to me as well.

**Pull request merged**